app/models/archived/intake/ctc_intake_2021.rb

```diff
@@ -198,7 +198,7 @@
 #  spouse_signature_pin_at                              :datetime
 #  spouse_suffix                                        :string
 #  spouse_tin_type                                      :integer
-#  spouse_was_blind                                     :integer          default(0), not null
```

Comment:
im betting that these migrations ran for the implementer before the enums were added, which is why me running the migrations here changed the values